Configuration/Emulator/task.md

```diff
@@ -11,22 +11,22 @@ of the device, simulate different network speeds, simulate rotation and other ha
 sensors, access the Google Play Store, and much more. Testing your app on the emulator is 
 faster and easier than doing so on a physical device. 
 
-To run the app on the emulator we will follow [this brief guide](https://developer.android.com/studio/run/managing-avds#createavd).
+To run the app on the emulator, we will follow [this brief guide](https://developer.android.com/studio/run/managing-avds#createavd).
 
  - Open the AVD manager (`Tools > AVD Manager`).
- - Click `Create Virtual Device`, at the bottom of the AVD Manager dialog.
- - Select a hardware profile, and then click `Next`.
- - Select the system image for a particular API level, and then click `Next`.
- - Change AVD properties as needed, and then click `Finish`.
+ - Click `Create Virtual Device` at the bottom of the AVD Manager dialog.
+ - Select a hardware profile and then click `Next`.
+ - Select the system image for a particular API level and then click `Next`.
+ - Change AVD properties as needed and then click `Finish`.
 
- Now your virtual device is set up and you can run your application from any 
- task by selecting the task you want to run in the **Edit Run/Debug Configurations**
- drop-down menu, as shown in the figure below.
+ Now your virtual device is set up and you can run any task of your application
```

Comment:
better keep the way it was ("run your application from any task") since we do not have multiple tasks in an application, but rather several applications in different tasks